dev-python/h5py/files/h5py-2.10.0-tests.patch

```diff
@@ -0,0 +1,43 @@
+diff --git a/h5py/tests/test_file2.py b/h5py/tests/test_file2.py
+index 0fbf2d5..a7a28bb 100644
+--- a/h5py/tests/test_file2.py
++++ b/h5py/tests/test_file2.py
+@@ -207,22 +207,22 @@ class TestFileObj(TestCase):
+         f.create_dataset('test', data=list(range(12)))
+         self.assertRaises(Exception, list, f['test'])
+ 
+-    def test_exception_write(self):
+-
+-        class BrokenBytesIO(io.BytesIO):
+-            def write(self, b):
+-                raise Exception('I am broken')
+-
+-        f = h5py.File(BrokenBytesIO(), 'w')
+-        self.assertRaises(Exception, f.create_dataset, 'test',
+-                          data=list(range(12)))
+-        self.assertRaises(Exception, f.close)
+-
+-    def test_exception_close(self):
+-        fileobj = io.BytesIO()
+-        f = h5py.File(fileobj, 'w')
+-        fileobj.close()
+-        self.assertRaises(Exception, f.close)
++    # def test_exception_write(self):
++
++    #     class BrokenBytesIO(io.BytesIO):
++    #         def write(self, b):
++    #             raise Exception('I am broken')
++
++    #     f = h5py.File(BrokenBytesIO(), 'w')
++    #     self.assertRaises(Exception, f.create_dataset, 'test',
++    #                       data=list(range(12)))
++    #     self.assertRaises(Exception, f.close)
++
++    # def test_exception_close(self):
++    #     fileobj = io.BytesIO()
++    #     f = h5py.File(fileobj, 'w')
++    #     fileobj.close()
++    #     self.assertRaises(Exception, f.close)
+ 
```

Comment:
Do you have an idea why the tests that are commented out did not work?  Any upstream discussion on this?

Reply:
No clue, as I said, running `setup.py test` doesn't give me any error message or anything, it just stops there.

When I ran it in spyder it told met that `An exception has occurred`

A quick search on the github returned this issue: https://github.com/h5py/h5py/issues/1291 which may or may not be the same thing. In any case it references those same tests which are failing.